pkg/controlplane/http/session.go

```diff
@@ -245,6 +246,99 @@ func (s *session) GetRunningUserAttributesSync(getRunningUserAttributesInput *v3
 	return &userNameOutput, err
 }
 
+// ReloadAppServicesConfigAndWaitForCompletion reloads the app service config in the backend and waits for job completion (blocking)
+func (s *session) ReloadAppServicesConfigAndWaitForCompletion(ctx context.Context, retryInterval, timeout time.Duration) error {
+	jobId, err := s.ReloadAppServicesConfig(ctx)
+	if err != nil {
+		return errors.Wrap(err, "Failed reloading app service config")
+	}
+
+	return s.WaitForJobCompletion(ctx, jobId, retryInterval, timeout)
+}
+
+// ReloadAppServicesConfig reloads the app service config in the backend (blocking)
+func (s *session) ReloadAppServicesConfig(ctx context.Context) (string, error) {
+	reloadAppServicesConfigInput := v3ioc.ReloadAppServicesConfigInput{
+		ControlPlaneInput: v3ioc.ControlPlaneInput{
+			Ctx: ctx,
+		},
+	}
+
+	reloadAppServicesConfigOutput := v3ioc.ReloadAppServicesConfigOutput{}
+
+	err := s.createResource(ctx,
+		"configurations/app_services/reloads",
+		"cluster_configuration_reload",
+		&reloadAppServicesConfigInput.ControlPlaneInput,
+		map[string]string{},
+		&reloadAppServicesConfigOutput.ControlPlaneOutput,
+		&reloadAppServicesConfigOutput.JobAttributes)
+
+	if err != nil {
+		return "", err
+	}
+
+	return reloadAppServicesConfigOutput.ID, nil
```

Comment:
Look like a mistake, it's an attribute called job_id

Reply:
It isn't a mistake, what's returned is:
```
{
    "id": "<id>",
    "type": "job",
}

Reply:
my, bad, cluster reloads still acts the "old" way and only it returns custom schema